testsuite/README

```diff
@@ -43,10 +43,9 @@ For questions and discussions for code development, join the developer list http
 
 .. badges
 
-.. |build| image:: https://travis-ci.com/MDAnalysis/mdanalysis.svg?branch=develop
-   :alt: Build Status
-   :target: https://travis-ci.com/MDAnalysis/mdanalysis
-
+.. |build| image:: https://github.com/MDAnalysis/mdanalysis/actions/workflows/gh-ci.yaml/badge.svg
+   :alt: Github Actions Build Status
+   :target: https://github.com/MDAnalysis/mdanalysis/actions/workflows/gh-ci.yaml
```

Comment:
Sorry this might be my fault, my intention here was that there should be an empty line after line this, but that there shouldn't be any whitespace on that line. Can you add the empty line back please?

Reply:
Ok. I have done the changes